OrbitCore/ElfFile.cpp

```diff
@@ -111,8 +111,7 @@ bool ElfFileImpl<ElfT>::IsAddressInTextSection(uint64_t address) const {
 }
 
 template <typename ElfT>
-bool ElfFileImpl<ElfT>::GetFunctions(Pdb* pdb,
-                                     std::vector<Function>* functions) const {
+bool ElfFileImpl<ElfT>::GetFunctions(Pdb* pdb) const {
```

Comment:
Rename it to Load/PopulateFunctions? It does not return them...

Reply:
done